### DISCUSSION TITLE
[BUG]初始化数据库报错

### DISCUSSION THREAD

**Discussion by BurtSweet** [dreamhunter2333/cloudflare_temp_email#389] [General]
## 复现步骤
初始化数据库报错了呢大佬
![error](https://github.com/user-attachments/assets/291393b3-de53-4cd9-afaa-46a9446105ea)

## 部署方式

- [ √]  用户界面部署

## 浏览器环境
chrome

**Comment by dreamhunter2333** [comment]
感谢反馈，此处是输入sql 文件的内容，而不是文件名
